VirtoCommerce.LiquidThemeEngine/Filters/FeatureFilter.cs

```diff
@@ -0,0 +1,65 @@
+namespace VirtoCommerce.LiquidThemeEngine.Filters
+{
+    using System;
+
+    using global::Scriban;
+
+    using Newtonsoft.Json.Linq;
+
+    public class FeatureFilter
+    {
+        public static string IsFeatureActive(TemplateContext context, string key, params object[] variables)
```

Comment:
You can return any Built-in .NET types from Liquid filters, you do not need to return only string then.

Reply:
see above